src/BPEssentials/Commands/Kit.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.ExtensionMethods.Cooldowns;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+using System.Linq;
+
+namespace BPEssentials.Commands
+{
+    public class Kit : Command
+    {
+        public void Invoke(ShPlayer player, string kit)
+        {
+            if (!Core.Instance.KitHandler.List.Any(x => x.Name == kit))
+            {
+                player.TS("expFileHandler_error_notFound", player.T(Core.Instance.KitHandler.Name), kit);
+                return;
+            }
+            var obj = Core.Instance.KitHandler.List.FirstOrDefault(x => x.Name == kit);
```

Comment:
Can be changed to `FirstOrDefault` only, as this will return `null` when not found.
```suggestion
            var obj = Core.Instance.KitHandler.List.FirstOrDefault(x => x.Name == kit);
            if (obj == null)
            {
                player.TS("expFileHandler_error_notFound", player.T(Core.Instance.KitHandler.Name), kit);
                return;
            }
```